libs/sdk-ui-charts/api/sdk-ui-charts.api.md

```diff
@@ -23,16 +22,18 @@ import { ITheme } from '@gooddata/sdk-backend-spi';
 import { IVisualizationCallbacks } from '@gooddata/sdk-ui';
 import { IVisualizationProps } from '@gooddata/sdk-ui';
 import { default as React_2 } from 'react';
+import { ValueOrPlaceholder } from '@gooddata/sdk-ui';
+import { ValuesOrPlaceholders } from '@gooddata/sdk-ui';
 import { VisType } from '@gooddata/sdk-ui';
 
 // @public
-export const AreaChart: import("react").ComponentType<IAreaChartProps>;
+export const AreaChart: (props: IAreaChartProps) => JSX.Element;
```

Comment:
just doublechecking that this change is compatible?

Reply:
Actually, it's a "breaking" change (React.ComponentType accepts also "children" prop, React.ComponentType could represent also class component), but I guess this signature without the children prop is more correct - our charts shouldn't accept children prop (it does nothing).

Reply:
ok, seems more like a bugfix then. let's keep it.